contest/contributions/pangea-Covid19.yml

```
@@ -0,0 +1,8 @@
+title: Pangea - COVID19 in Italy
+author: Pangea
```

We expect `author` to be a person's name, put you can of course keep Pangea
```suggestion
author: Andrea Romualdi, Pangea
```